lab1.py

```diff
@@ -0,0 +1,50 @@
+
+#---------------------Task 1-------------------
+
+number1 = input("please enter start number: ")
+number2 = input("please enter end number: ")
+
+start = int(number1)
+end = int(number2)
+test = start
+
+for i in range(start, end+1):
+    if i > 1:
+        for j in range(2, i):
+            if(i % j == 0):
+                break
+        else:
+            print(i)
+
+
+#---------------------Task 2-------------------
+
+name = input("please enter name: ")
+gpa = input("please enter GPA: ")
+
+
+def show_student(name, gpa=0):
+   
+    print("your name: ", name)
+
+    if gpa:
+         print("your GPA: ", gpa)
+    else:
+        print("your GPA: 0")
```

Comment:
Good job Abdullah 👍🏽

No need for these lines if you assign a default value to the argument in the function definition `(name, GPA=0)`
So it will take the default value in the definition if you do not pass it.